src/main/java/study/lotto/model/Lotto.java

```diff
@@ -35,6 +36,15 @@ public static Lotto of(List<LottoNumber> lottoNumberList) {
         return new Lotto(lottoNumberList);
     }
 
+    public static Lotto of(String[] lottoNumbers) {
+        List<LottoNumber> lottoNumberList = Arrays.stream(lottoNumbers)
+                .map(Integer::parseInt)
+                .map(LottoNumber::of)
+                .collect(Collectors.toList());
+
+        return new Lotto(lottoNumberList);
```

Comment:
```suggestion
        return Arrays.stream(lottoNumbers)
                .map(Integer::parseInt)
                .map(LottoNumber::of)
                .collect(Collectors.collectingAndThen(Collectors.toList(), Lotto::new));
```
위와 같이 사용하는 방법도 있겠네요!